packages/zowe-explorer/src/uss/init.ts

```diff
@@ -0,0 +1,173 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the *
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at *
+ * https://www.eclipse.org/legal/epl-v20.html                                      *
+ *                                                                                 *
+ * SPDX-License-Identifier: EPL-2.0                                                *
+ *                                                                                 *
+ * Copyright Contributors to the Zowe Project.                                     *
+ *                                                                                 *
+ */
+
+import * as globals from "../globals";
+import * as vscode from "vscode";
+import * as ussActions from "./actions";
+import * as refreshActions from "../shared/refresh";
+import { IZoweUSSTreeNode, IZoweTreeNode, IZoweTree } from "@zowe/zowe-explorer-api";
+import { Profiles } from "../Profiles";
+import * as contextuals from "../shared/context";
+import { getSelectedNodeList } from "../shared/utils";
+import { createUSSTree } from "./USSTree";
+import { initSubscribers } from "../shared/init";
+
+export async function initUSSProvider(context: vscode.ExtensionContext) {
```

Comment:
Do we want to make this a default export since its the only exported function in this file?

Reply:
no strong preference.
I guess I do like [this comment on stackoverflow](https://stackoverflow.com/a/33307487/16555983) about code-refactoring made easier if using named exports ( no `default` keyword) 😋